draft-ietf-tls-esni.md

```diff
@@ -626,21 +626,23 @@ usual, authenticating the connection for the true server name.
 If the server used ClientHelloOuter, the client proceeds with the handshake,
 authenticating for ECHConfig.contents.public_name as described in
 {{auth-public-name}}. If authentication or the handshake fails, the client MUST
-return a failure to the calling application. It MUST NOT use the retry keys.
+return a failure to the calling application. It MUST NOT use the retry
+configurations.
 
-Otherwise, when the handshake completes successfully with the public name
-authenticated, the client MUST abort the connection with an "ech_required"
-alert. It then processes the "retry_configs" field from the server's
-"encrypted_client_hello" extension.
+Otherwise, if the both authentication and the handshake complete successfully,
```

Comment:
Nit: s/the both/both/

Reply:
Huh, that's new. I fixed it in https://github.com/davidben/draft-ietf-tls-esni/tree/retry-config-naming, but GitHub doesn't seem to have noticed. Hopefully it's just some weird slowness on their end and the PR will notice sooner or later?

Reply:
Okay, now it's noticed. Done.